pyramid_debugtoolbar/panels/request_vars.py

```diff
@@ -6,6 +6,66 @@
 
 _ = lambda x: x
 
+
+# extractable_request_attributes allow us to programmatically pull data
+# the format is...
+#                                 ( attr_, is_show_dict, is_execute_method )
+extractable_request_attributes = (
+                                  ('accept_charset', None, None),
+                                  ('accept_encoding', None, None),
+                                  ('accept_language', None, None),
+                                  ('application_url', None, None),
+                                  ('authenticated_userid', None, None),
+                                  ('authorization', None, None),
+                                  ('cache_control', None, None),
+                                  ('context', None, None),
+                                  ('current_route_path', None, True),
+                                  ('current_route_url', None, True),
```

Comment:
Makes sense.  Done.

The API docs for those 2 items should probably reference this with something like "If you simply want to read the current value, you might just want to use X".